webpack.config.js

```diff
@@ -111,8 +111,9 @@ module.exports = ({ stats = false } = {}) => ({
     alias: {
       'react-dom': '@hot-loader/react-dom',
       'bn.js': path.resolve(__dirname, 'node_modules/bn.js'),
+      'react-inspector': path.resolve(__dirname, 'node_modules/react-inspector'),
     },
-    modules: ['custom', 'src', 'node_modules'],
+    modules: [path.resolve(__dirname, 'custom'), path.resolve(__dirname, 'src'), 'node_modules'],
```

Comment:
That's strange too

Reply:
That's actually the recommended way now that I check [resolve.modules](https://webpack.js.org/configuration/resolve/#resolvemodules:~:text=path.resolve(__dirname%2C%20'src')%2C)